packages/@okta/i18n/src/properties/login.properties

```diff
@@ -489,6 +490,7 @@ oktaverify.timeout = Your push notification has expired.
 oktaverify.warning = Haven't received a push notification yet? Try opening the Okta Verify App on your phone.
 oktaverify.numberchallenge.instruction = On your phone, tap <span class="challenge-number">{0}</span> on Okta Verify prompt to continue.
 oktaverify.numberchallenge.explain = This extra step helps us make sure it's really you signing in.
+oktaverify.button.label = Sign in using Okta Verify
```

Comment:
where is this label used ?

Reply:
Removed it, it's not needed after rebasing my master.